src/gui/clips/SampleClipView.cpp

```diff
@@ -346,33 +346,33 @@ void SampleClipView::setAutomationGhost()
 }
 
 //! Split this Clip.
-/*! \param pos the position of the split, relative to the start of the clip */
-bool SampleClipView::splitClip( const TimePos pos )
+/**
```

Comment:
This comment could also be removed if you update the comment in the `ClipView` header